TodoList/Models/Item.swift

```diff
@@ -7,3 +19,20 @@ struct Item {
         self.isComplete = isComplete
     }
 }
+//NOTE: UserDefaults is a temporary solution
+extension Todos {
+    func save() {
+        if let encoded = try? JSONEncoder().encode(self) {
+            UserDefaults.standard.set(encoded, forKey: "todoList")
+        }
+    }
+
+    static func load() -> Todos {
+        if let savedItems = UserDefaults.standard.object(forKey: "todoList") as? Data {
+            if let loadedItems = try? JSONDecoder().decode(Todos.self, from:savedItems) {
+                return loadedItems
+            }
+        }
+        return Todos(items: [])
+    }
+}
```

Comment:
How do you all feel about breaking up `Todos` and `Item` into separate files?

Reply:
Works for me, since is likely to grow.

Reply:
Then for sure, please do break them apart.